crates/host/src/module/wasmer_sys.rs

```diff
@@ -26,9 +26,9 @@ pub(crate) fn make_engine() -> Engine {
 
     // the only place where the wasm compiler engine is set
     #[cfg(feature = "wasmer_sys_dev")]
```

Comment:
I wish I'd named these feature flags differently. If we're doing a major upgrade of wasmer, maybe I should come and do that as a follow up PR. These are both valid production compilers, but with different characteristics.